src/benchmarks/micro/libraries/System.Collections/Dictionary/DictionarySequentialKeys.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.Collections.Immutable;
```

Comment:
the existing benchmarks that you can find in this file has been added in https://github.com/dotnet/performance/pull/938 to validate https://github.com/dotnet/coreclr/pull/27299 which required a very specific benchmark coverage of the `Dictionary<TKey, Value>` APIs. We don't need such detailed coverage for other dictionary types as they already have good overall coverage. Please revert all the changes in this file.